alarm/models.py

```diff
@@ -14,26 +14,44 @@ class Phone(models.Model):
     user = models.ForeignKey(User, on_delete=models.CASCADE)
     number = PhoneNumberField(blank=True, unique=True, region='UA')
     enabled = models.BooleanField(default=False)
+    message = models.TextField(default=None)
 
     def __str__(self):
         return str(self.number)
 
+    def refresh_message(self, message):
+        self.message = message
+        self.save()
+
+    def clear_old_message(self):
+        self.message = None
+        self.save()
+
 
 class Threshold(models.Model):
     phone = models.ForeignKey(Phone, on_delete=models.CASCADE)
     trade_pair = models.CharField(max_length=255)
     price = models.DecimalField(max_digits=10, decimal_places=2)
 
     def __str__(self):
-        return f"{self.trade_pair}: {self.price}"
+        return f"{self.price}"
 
     def clean(self):
         super().clean()
-        valid_trade_pair = get_binance_trade_pairs()
-        if self.trade_pair not in valid_trade_pair:
+        valid_list_of_trade_pairs = get_binance_valid_list_of_trade_pairs()
```

Comment:
Is is important here to emphasise a type of a returned value?
I'd rename to `valid_trade_pairs` and `get_binance_valid_trade_pairs`.

Reply:
++